app/api/tool_utilities.py

```diff
@@ -1,8 +1,8 @@
 import json
 import os
-from app.services.logger import setup_logger
-from app.services.tool_registry import ToolFile
-from app.api.error_utilities import VideoTranscriptError, InputValidationError, ToolExecutorError
+from services.logger import setup_logger
```

Comment:
Need to keep the `app.` prefix since we are using the parent to define the relative paths for the project

Note how we start using the `./local-start.sh`

Reply:
added these prefixes to all files